meshmode/array_context.py

```diff
@@ -140,7 +134,7 @@ def is_array_container(ary: object):
 
 
 @is_array_container.register(ArrayContainer)
-def _(ary: ArrayContainer):
+def _is_array_container_ac(ary: ArrayContainer):
```

Comment:
FYI: Changed because `_` as a function name leads to stack frames that aren't very readable.